cli/features/steps/steps.ls

```diff
@@ -130,9 +130,9 @@ module.exports = ->
       jsdiff-console actual-content.to-string!trim!, expected-content.trim!, done
 
 
-  @Then /^the full command "([^"]*)" is executed$/ timeout: 5_000, (command, done) ->
+  @Then /^the full command "([^"]*)" is executed$/ timeout: 20_000, (command, done) ->
```

Comment:
I've been running into repeated timeouts with this setting on my machine. Downloading the Docker images can take a while. Let's go with 60 seconds here.